static/app/views/starfish/views/spans/spanTimeCharts.tsx

```diff
@@ -189,7 +192,7 @@ export const getSpanTotalTimeChartQuery = (
   const validConditions = conditions.filter(Boolean);
 
   return `SELECT
-    count() AS throughput,
+    divide(count(), multiply(12, 60)) as spm,
```

Comment:
Maybe factor out the interval into a constant or pass it in as a param, but we can do that later